src/lib.rs

```diff
@@ -880,14 +886,42 @@ impl Output {
 #[derive(Debug, Eq, PartialEq)]
 pub struct Value(value::Value);
 
+impl AsRef<u64> for Value {
+    fn as_ref(&self) -> &u64 {
+        &self.0.as_ref()
+    }
+}
+
+impl From<u64> for Value {
+    fn from(number: u64) -> Value {
+        value::Value(number).into()
+    }
+}
+
 #[wasm_bindgen]
 impl Value {
-    pub fn from_u64(number: u64) -> Self {
-        Value(value::Value(number))
+    pub fn from_str(s: &str) -> Result<Value, JsValue> {
```

Comment:
Could you document that this is due to JS and u64 support issues?

Reply:
done? I think, at least if you meant in the code docs